src/components/DonateButton/DonateButton.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import styles from './DonateButton.module.css';
+
+function Donatebutton(props) {
+  return (
+    <a
+      className={styles.text}
+      href="http://www.vg.no"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      <div className={styles.Boxed}>
+        <b>Doner til årets formål</b>
+      </div>
+    </a>
+  );
+}
```

Comment:
Dette er veldig bra. Men dette kan vi generalisere etter hvert. Vi kan bytte navn til bare Button.

Og så kan vi gjøre om slik at det er

```
function Button({name, link}) {
return (
<a
className={styles.text}
href={link}
target="_blank"
rel="noopener noreferrer"
>
<div className={styles.Boxed}>
<b>{text}</b>
</div>
</a>
);
}
```

På den måten kan man jo bruke knappen til hva man vil. Og så når man kaller på den i kode så bare sender man inn link og tekst. Men bra sånn som det er nå. Så kan vi heller utvide etter hvert.

Er egentlig litt sånn man ønsker at det skal være. Ikke lag ting for generelt. Bare lag det sånn at det virker. Og så når man begynner å kopiere ting flere ganger hit og dit så er det bedre å lage det generelt.

Reply:
Det hørtes jo veldig kult ut!